spec/helpers/application_helper/buttons/vm_timeline_spec.rb

```diff
@@ -1,6 +1,13 @@
+require 'shared/helpers/application_helper/buttons/timeline'
+
 describe ApplicationHelper::Button::VmTimeline do
+  include_context 'ApplicationHelper::Button::Basic'
+  let(:sandbox) { Hash.new }
+  let(:instance_data) { {'record' => record} }
+  let(:props) { Hash.new }
   let(:record) { FactoryGirl.create(:vm) }
-  let(:button) { described_class.new(setup_view_context_with_sandbox({}), {}, {'record' => record}, {}) }
 
-  it_behaves_like 'a timeline button', :entity => 'VM'
+  describe '#calculate_properties' do
+    include_context 'ApplicationHelper::Button::Timeline#calculate_properties', 'VM'
```

Comment:
Timeline buttons have a specific error message that vary only in the entity. The entity is passed on with the argument. How should the context be written to reflect that?